pipelines/jobs/configurations/jdk17u_release.groovy

```diff
@@ -17,6 +17,9 @@ targetConfigurations = [
         'ppc64leLinux': [
                 'temurin'
         ],
+        'ppc64Aix'    : [
```

Comment:
should we build AIX for jdk17 /19? if we do not officially release it out

Reply:
otherwise i am happy to approve this

Reply:
We want to officially release it, but had an issue in Temurin-compliance that was blocking it (which was why it was not added originally).  Think Stewart & Tyler have arrived at a solution to the blocking issue, so we can reconstitute it.